ext/openssl/xp_ssl.c

```diff
@@ -2015,24 +1971,6 @@ static int php_openssl_enable_crypto(php_stream *stream,
 				n = -1;
 			} else {
 				sslsock->ssl_active = 1;
-
-				if (PHP_STREAM_CONTEXT(stream)) {
-					zval *val;
-					if (NULL != (val = php_stream_context_get_option(PHP_STREAM_CONTEXT(stream),
-						"ssl", "capture_session_meta"))
-					) {
-						 php_error(E_DEPRECATED,
```

Comment:
Should we still retain a warning here to say that capture_session_meta is no longer supported and will be ignored?

Reply:
I don't remember us doing something like this? But could do it.